common/constants.go

```diff
@@ -828,3 +828,9 @@ const GenesisStorageSuffix = "_genesis"
 
 // MetricAccountsSnapshotNumNodes is the metric that outputs the number of trie nodes written for accounts after snapshot
 const MetricAccountsSnapshotNumNodes = "erd_accounts_snapshot_num_nodes"
+
+// MetricAccountsSyncBytesReceived is the metric that outputs the number of bytes received for accounts during sync
+const MetricAccountsSyncBytesReceived = "erd_accounts_sync_bytes_received"
```

Comment:
AccountsSyncBytesReceived => TrieSyncNumReceivedBytes
AccountsSyncNodesProcessed => TrieSyncNumProcessedNodes

apply these renamings to the entire PR

Reply:
done